cmd/keygenerator/p2pConverter.go

```diff
@@ -0,0 +1,43 @@
+package main
+
+import (
+	libp2pCrypto "github.com/libp2p/go-libp2p-core/crypto"
+	"github.com/libp2p/go-libp2p-core/peer"
+)
+
+type p2pConverter struct{}
+
+// NewP2pConverter creates a new instance of p2p converter
+func NewP2pConverter() *p2pConverter {
+	return &p2pConverter{}
+}
+
+// Len return zero
+func (p *p2pConverter) Len() int {
+	return 0
+}
+
+// Decode does nothing
+func (p *p2pConverter) Decode(humanReadable string) ([]byte, error) {
+	return nil, nil
```

Comment:
Might  return ErrNotImplemented or something.

Reply:
added not implemented error on return